test/testing/test_pytest_plugin.py

```diff
@@ -24,6 +25,34 @@ def test_rabbit_manager(rabbit_manager):
     assert "/" in [vhost['name'] for vhost in rabbit_manager.get_all_vhosts()]
 
 
+def test_rabbit_config_random_vhost(testdir):
+
+    testdir.makepyfile(
+        """
+        import re
+
+        def test_rabbit_config(rabbit_config):
+            assert re.search("test_[a-z]+$", rabbit_config['AMQP_URI'])
+        """
+    )
+    result = testdir.runpytest()
+    assert result.ret == 0
+
+
+def test_rabbit_config_specific_vhost(testdir):
+
+    testdir.makepyfile(
+        """
+        def test_rabbit_config(rabbit_config):
+            assert "specified_vhost" in rabbit_config['AMQP_URI']
+        """
+    )
+    result = testdir.runpytest(
+        "--amqp-uri", "amqp://guest:guest@localhost:5672/specified_vhost"
+    )
+    assert result.ret == 0
```

Comment:
i'm unfamiliar with this; what is the `.ret` telling you? is a negative assertion strong enough to tell us that this test is doing something useful?